crates/task/src/task_template.rs

```diff
@@ -184,8 +184,12 @@ impl TaskTemplate {
             .context("hashing task variables")
             .log_err()?;
         let id = TaskId(format!("{id_base}_{task_hash}_{variables_hash}"));
+
+        let mut env = cx.project_env.clone();
+        env.extend(self.env.iter().map(|(k, v)| (k.to_owned(), v.to_owned())));
```

Comment:
I wonder if 
```suggestion
        env.extend(self.env.clone().into_iter());
```
could work too.

Reply:
Works!